packages/dashboard/src/components/RegisterFrame/ColumnContentLayout.tsx

```diff
@@ -0,0 +1,82 @@
+import { experimentalStyled as styled } from '@material-ui/core/styles'
+import { getMaskColor } from '@masknet/theme'
+import { InfoOutlined } from '@material-ui/icons'
+import { Button, Typography } from '@material-ui/core'
+import { useNavigate } from 'react-router'
+
+const Container = styled('div')`
```

Comment:
I recommend to use `makeStyles`

Reply:
@nuanyang233 plz re-read the https://github.com/DimensionDev/Maskbook/blob/develop/docs/css-in-js.md, thanks

Reply:
Here is an update about this guide and the current usage is allowed actually (cause it is simple)